ppr-ui/src/components/parties/Debtors.vue

```diff
@@ -70,14 +70,18 @@
               :key="row.item.id"
               class="debtor-row"
             >
-              <td class="list-item__title title-text">
+              <td class="list-item__title title-text pl-7">
                 <v-row no-gutters>
-                  <v-col cols="12" style="padding-left: 14px;">
-                    <div class="icon-div mt-n1 pr-4">
+                  <v-col cols="3">
+                <div class="icon-div mt-n1 pr-4">
```

Comment:
indent within the v-col here and below please 🙏

Reply:
done